theforeman.org/pipelines/test/testKatello.groovy

```diff
@@ -3,14 +3,14 @@ def katello_versions = [
         'foreman': 'develop',
         'ruby': ['2.7']
     ],
+    'KATELLO-4.1': [
+        'foreman': '2.5-stable',
+        'ruby': ['2.7']
+    ],
     'KATELLO-4.0': [
         'foreman': '2.4-stable',
         'ruby': ['2.5']
     ],
-    'KATELLO-3.18': [
-        'foreman': '2.3-stable',
-        'ruby': ['2.5']
-    ],
```

Comment:
I'd keep this in, at least until we really drop 3.18. It doesn't hurt to have a slightly bigger table.

Reply:
3.18 shouldn't be dropped until 4.1 is actually GA

Reply:
Thanks, updated to restore 3.18.